spec/models/miq_queue_spec.rb

```diff
@@ -336,6 +336,37 @@
       expect(MiqQueue.get).to eq(nil)
     end
 
+    it "should accept non-Array args (for now)" do
+      class MiqQueueSpecNonArrayArgs
```

Comment:
I think we shouldn't create classes that persist across specs.  Perhaps this should be created in a before and removed in an after block.